x/mongo/driver/batch_cursor.go

```diff
@@ -98,11 +99,12 @@ func NewCursorResponse(response bsoncore.Document, server Server, desc descripti
 
 // CursorOptions are extra options that are required to construct a BatchCursor.
 type CursorOptions struct {
-	BatchSize      int32
-	MaxTimeMS      int64
-	Limit          int32
-	CommandMonitor *event.CommandMonitor
-	Crypt          *Crypt
+	BatchSize        int32
+	MaxTimeMS        int64
+	Limit            int32
+	CommandMonitor   *event.CommandMonitor
+	Crypt            *Crypt
+	ServerAPIOptions *ServerAPIOptions
```

Comment:
nit: rename this field to `ServerAPI` for consistency with the other types you've modified.

Reply:
Sounds good!